modules/learn/pages/security/on-the-wire-security.adoc

```diff
@@ -13,9 +13,17 @@ The interfaces provided by Couchbase Server for configuring on-the-wire security
 This can be disabled, for the whole cluster; over either HTTP or HTTPS, or over both.
 
 * _Establishing the Cluster Encryption-Level_.
-The level of encryption imposed on inter-node communications within the cluster can be configured: either as _control_ (meaning that only server-management information is passed in encrypted form) or _all_ (meaning that all information, including data handled by services, is passed in encrypted form).
+The level of encryption imposed on inter-node communications within the cluster can be configured.
+This can be:
+
+** `control`, meaning that server-management information passed between nodes is passed in encrypted form.
+
+** `all`, meaning that all information passed between nodes, including data handled by services, is passed in encrypted form.
+
+** `strict`, meaning the same as `all`, but meaning additionally that only encrypted communication is permitted between nodes and between the cluster and external clients.
```

Comment:
Can we not have "meaning the same as `all`" ? Same as all provides a different understanding. 
Instead it is `all` plus only encrypted communication is permitted.

Reply:
Thanks - although I may not quite understand the best rewording. So, perhaps: " `strict`, meaning `all`, but with only encrypted communication permitted...etc" .. something like that?

Reply:
I would leave the "but" out. 
" strict, meaning all with only encrypted communication permitted...etc"